doc/protocol/README.md

```diff
@@ -1,9 +1,7 @@
 # KUKSA.val Protocol
+KUKSA.val supports parts of VISS V1 https://www.w3.org/TR/vehicle-information-service/ as well as parts of the upcomming VISS2 standard ([Gen2 Core](https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Core.html), [Gen2 Transport](https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Transport.html)), that are applicable to in-vehicle VSS servers. KUKSA.val also has a gRPC interface to communicate.
 
-KUKSA.val supports VISS V1 https://www.w3.org/TR/vehicle-information-service/ and extensions as well as parts of the upcomming VISS2 standard ([Gen2 Core](https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Core.html), [Gen2 Transport](https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Transport.html)), that are applicable to in-vehicle VSS servers. 
+The following document will detail, what is supported on the websocket (aka KUKSA.val server) and gRPC Interface (aka data broker):
 
-The following documents will detail, what is supported on the Websocket Interface:
 
-
-## [get calls](get.md)
```

Comment:
So what is the current plan - remove the links to get/set-documents but keep the links? Or better to remove the documents as well, or keep the links, to get consistency.

Reply:
I also thought of this. But I think since these files are basically copies of the w3c doc here: https://www.w3.org/TR/vehicle-information-service/#message-structure I think we could drop them.